graphql_compiler/compiler/context_helpers.py

```diff
@@ -1,3 +1,4 @@
+# Copyright 2017 Kensho Technologies, Inc.
```

Comment:
i think you could add a travis check for this -- some bash 1-liner with `find . -iname='*.py'` and `head -n1` ?

Reply:
Good idea, I'll try to do it in a separate PR.